cpu/nrf52/periph/Kconfig

```diff
@@ -19,5 +19,13 @@ config MODULE_PERIPH_SPI
 config MODULE_SAUL_NRF_VDDH
     bool "Internal Voltage Sensor"
     depends on HAS_PERIPH_ADC
+    select MODULE_PERIPH_ADC
 
 endif # TEST_KCONFIG
+
+config HAVE_SAUL_NRF_VDDH
+    bool
+    select MODULE_SAUL_NRF_VDDH if MODULE_SAUL_DEFAULT && TEST_KCONFIG
```

Comment:
`&& TEST_KCONFIG` is not required, as `MODULE_SAUL_DEFAUL` already depends on it.